### ISSUE TITLE
Hi I have a questions about experiments

### ISSUE THREAD

**Issue by szu2020222001** [Woodman718/CapsNets#1]
Thanks for good research and i'm interested in skin cancer lession classification task. i want to ask some questions. How to get the data set for this experiment？What do the different compressed packages of this website "https://aistudio.baidu.com/aistudio/datasetdetail/151696/0" represent?

**Comment by Woodman718** [comment]
Thank you for your interest in our work. The dataset is split into three sets: training, validation, and test sets. The training and validation sets are utilized for training purposes, while the test set is exclusively reserved for inference or evaluation.
We recommend registering an account on Aistudio to obtain the processed data from the latest link below. Additionally, we suggest creating an account on Techrxiv to download the preprint.
Dataset：https://aistudio.baidu.com/aistudio/datasetdetail/218024
Preprint：https://www.techrxiv.org/articles/preprint/A_Novel_Skin_Cancer_Assisted_Diagnosis_Method_based_on_Capsule_Networks_with_CBAM/23291003/1
